apps/api/src/modules/summary/summary.service.ts

```diff
@@ -27,9 +33,11 @@ export class SummaryService extends CoreService<Summary> {
     if (inputFile) {
       const { createReadStream, filename } = await inputFile;
 
-      const uniqueIdentifier = uuidv4();
+      const uniqueIdentifier = uuidv4().replace(/-/g, '').substring(0, 12);
       const modifiedFilename = `${uniqueIdentifier}_${filename}`;
-      const fileLocation = join(process.cwd(), `./src/upload/${modifiedFilename}`);
+      const fileLocation = join(process.cwd(), configService.getOrThrow('Upload_Path'), modifiedFilename);
```

Comment:
We won't use cwd.
We directly use the path from env value.
You can use process.cwd() + /upload as default folder though.

Reply:
Added the changes, removed process.cwd()

Reply:
Please do all the changes like asked above.

You did not do
- You can use process.cwd() + /upload as default folder though.

Reply:
Added process.cwd() + /upload as fallback condition